source/templates.tex

```diff
@@ -3344,7 +3344,7 @@
 
 \item
 The template parameter list of a specialization shall not contain default
-template argument values.\footnote{There is no way in which they could be used.}
+template argument values.\footnote{There is no way in which they can be used.}
```

Comment:
Ugh, ISO's rules are awful. I think this doesn't really make sense, because such default arguments can't even exist, so of course they can't be used. How about:
```suggestion
template argument values.\footnote{There is no context in which they would be used.}
```